pkg/commands/apply/apply.go

```diff
@@ -43,16 +43,16 @@ const (
 	CommentPrefix       = "**`🔱 Guardian 🔱 APPLY`** -"
 )
 
-var _ cli.Command = (*ApplyRunCommand)(nil)
+var _ cli.Command = (*ApplyCommand)(nil)
 
 // RunResult is the result of a apply operation.
 type RunResult struct {
 	commentDetails string
 }
 
-// ApplyRunCommand is a subcommand of apply and implements the cli.Command interface.
+// ApplyCommand is a subcommand of apply and implements the cli.Command interface.
```

Comment:
nit: this comment no longer makes sense...

Reply:
fixed.